src/NumericalAlgorithms/LinearOperators/PowerMonitors.cpp

```diff
@@ -17,6 +17,39 @@
 
 namespace PowerMonitors {
 
+double detail::relative_truncation_error_impl(
+    const DataVector& input_power_monitors, const size_t upperBound) {
+  // upperBound needs to be smaller than the number of power monitors in the
+  // current dimension, i.e.
+  // upperBound < gsl::at(input_power_monitors, sliced_dim).size()
+
+  // Compute weighted average and total sum in the current dimension
+  double weighted_average = 0.0;
+  double weight_sum = 0.0;
+  double weight_value = 0.0;
+  for (size_t index = 0; index < upperBound; ++index) {
+    // Compute current weight
+    weight_value = exp(-square(
+        index - static_cast<double>(upperBound) + 0.5)
+        );
```

Comment:
Run clang-format over this file please. Easiest is to just run it like this when you have staged your changes in git:

```
$ git-clang-format
```